index.html

```diff
@@ -1,44 +1,225 @@
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="1">See Project</button>
+                </div>
+            </div>
+            <div class="works-2">
+                <div class="project-image"><img class="image" src="images/header/second.png" alt="my first project"></div>
+                <div class="projects">
+                    <h2 class="project-tittle">Multi-Post Stories</h2>
+                    <ul class="project-info">
+                        <li class="client">FACEBOOK</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>Full Stack Dev</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>2015</li>
                     </ul>
-                </section>
+                    <p class="primary-text">Experimental content creation feature that allows users to add to an
+                        existing
+                        story over the course of a day without spamming their friends.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="2">See Project</button>
+                </div>
+            </div>
+            <div class="works">
+                <div class="project-image"><img class="image" src="images/header/third.png" alt="my first project"></div>
+                <div class="projects">
+                    <h2 class="project-tittle">Facebook 360</h2>
+                    <ul class="project-info">
+                        <li class="client">FACEBOOK</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>Full Stack Dev</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="primary-text">Exploring the future of media in Facebook's first Virtual Reality app; a
+                        place
+                        to discover and enjoy 360 photos and videos on Gear VR.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="3">See Project</button>
+                </div>
+            </div>
+            <div class="works-2">
+                <div class="project-image"><img class="image" src="images/header/desktop/d.png" alt="my first project"></div>
```

Comment:
- [ ] Kindly fix the path to this image, it's not showing 👍 

![image](https://user-images.githubusercontent.com/50721479/194709110-8e33b283-2e6e-432b-9a5b-3064657fbc6f.png)